lib/src/changes.dart

```diff
@@ -96,6 +96,108 @@ final Map<String, String> _textReplacements = {
   'Endianness.HOST_ENDIAN': 'Endian.host',
   'Int8List.BYTES_PER_ELEMENT': 'Int8List.bytesPerElement',
   // and, other (long tail) dart:typed_data renames omitted
+
+  // dart:io/data_transformer.dart
+  'ZLIB': 'zlib',
+  'GZIP': 'gzip',
+  'ZLibOption.MIN_WINDOW_BITS': 'ZLibOption.minWindowBits',
+  'ZLibOption.MAX_WINDOW_BITS': 'ZLibOption.maxWindowBits',
+  'ZLibOption.DEFAULT_WINDOW_BITS': 'ZLibOption.defaultWindowBits',
+  'ZLibOption.MIN_LEVEL': 'ZLibOption.minLevel',
+  'ZLibOption.MAX_LEVEL': 'ZLibOption.maxLevel',
+  'ZLibOption.DEFAULT_LEVEL': 'ZLibOption.defaultLevel',
+  'ZLibOption.MIN_MEM_LEVEL': 'ZLibOption.minMemLevel',
+  'ZLibOption.MAX_MEM_LEVEL': 'ZLibOption.maxMemLevel',
+  'ZLibOption.DEFAULT_MEM_LEVEL': 'ZLibOption.defaultMemLevel',
+  'ZLibOption.STRATEGY_FILTERED': 'ZLibOption.strategyFiltered',
+  'ZLibOption.STRATEGY_HUFFMAN_ONLY': 'ZLibOption.strategyHuffmanOnly',
+  'ZLibOption.STRATEGY_RLE': 'ZLibOption.strategyRle',
+  'ZLibOption.STRATEGY_FIXED': 'ZLibOption.strategyFixed',
+  'ZLibOption.STRATEGY_DEFAULT': 'ZLibOption.strategyDefault',
+
+  // dart:io/file.dart
+  'FileMode.READ': 'FileMode.read',
```

Comment:
The top-levels READ, WRITE, APPEND, WRITE_ONLY, and WRITE_ONLY_APPEND should be moved to FileMode.read, etc..

Reply:
I omitted these; we're not using analysis server refactoring to perform these updates - we'd need more info in the deprecated annotations for that (https://github.com/dart-lang/sdk/issues/32726). We use the analysis server to locate deprecation warnings and then apply a heuristic; top level `READ`, `WRITE`, `APPEND`, ... seemed too likely to have false positives. I'll add a comment about that.